sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -841,7 +842,7 @@ struct _pi_queue : _pi_object {
   // PI queue is in general a one to many mapping to L0 native queues.
   struct pi_queue_group_t {
     pi_queue Queue;
-    pi_queue_group_t() = delete;
+    // pi_queue_group_t() = delete;
```

Comment:
why removed? I gauss it's the requirement of std::map but then let's define it so it is not leaving the created object in uninitialized state

Reply:
Restored.